### PR TITLE
add blog link with how-to-guide

### PR DESCRIPTION
**Pull request by fiquick**
Hi, there's is a really nice blog article with how to set up Private Link w. Amazon - I'd like to link to it in the Aura docs

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-363.surge.sh

Updated pages:
- [Secure connections](https://neo4j-docs-aura-363.surge.sh/aura/platform/security/secure-connections)

File: modules/ROOT/pages/platform/security/secure-connections.adoc [hunk 1]
```diff
@@ -41,6 +41,8 @@ AuraDB Enterprise supports private endpoints on AWS using https://aws.amazon.com
 
 Once activated, you can create an endpoint in your VPC that connects to Aura.
 
+For a step-by-step guide, see the https://neo4j.com/blog/neo4j-aws-privatelink-configuration/[How to Configure Neo4j Aura With AWS PrivateLink] blog article.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/platform/security/secure-connections.adoc:44]:
```suggestion
For a step-by-step guide, see the https://neo4j.com/blog/neo4j-aws-privatelink-configuration/[How to Configure Neo4j Aura With AWS PrivateLink] blog article.
```
```suggestion
For a step-by-step guide, see the link:https://neo4j.com/blog/neo4j-aws-privatelink-configuration/[How to Configure Neo4j Aura With AWS PrivateLink] blog article.
```

**Comment by fiquick** [comment]
[Aura: SSO Azure AD is now MS Entra ID](https://trello.com/c/6qcNWozO/325-aura-sso-azure-ad-is-now-ms-entra-id)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-363.surge.sh

Updated pages:
- [Secure connections](https://neo4j-docs-aura-363.surge.sh/aura/platform/security/secure-connections)

**Review by AlexicaWright — APPROVED** [review]

**Pull request merged**